util/util.go

```diff
@@ -30,6 +32,54 @@ func StrToSemVer(version string) (*semver.Version, error) {
 	return v, nil
 }
 
+func SemVerMatchRange(k8sVersion, versionRange string) (bool, error) {
+	toMatch, err := sv.Make(k8sVersion[1:])
+	if err != nil {
+		return false, fmt.Errorf("Cluster version [%s] can not be parsed as semver: %v", k8sVersion, err)
+	}
```

Comment:
I could be wrong here, but are we checking the length? It could be possible to have the `k8sVersion[1:]` result in a non-zero length right?

Reply:
We have `validateVersion` in `cluster/validation.go` to validate this, I'm fine with adding another check in case of better safe than sorry.

Reply:
That might be a good idea even if it's not "possible". We could also write a common "split" utility that the two locations share!